test/modules/TestResultCoordinator/Reports/DirectMethod/Connectivity/DirectMethodConnectivityReportGenerator.cs

```diff
@@ -162,24 +157,11 @@ public async Task<ITestResultReport> CreateReportAsync()
                     $"{dmSenderTestResult.GetFormattedResult()}. ReceiverTestResult: {dmReceiverTestResult.GetFormattedResult()}");
             }
 
-            bool didFindMatch = false;
             if (dmSenderTestResult.SequenceNumber == dmReceiverTestResult.SequenceNumber)
             {
-                dmReceiverTestResult = JsonConvert.DeserializeObject<DirectMethodTestResult>(receiverTestResults.Current.Result);
```

Comment:
I am not familiar with the code but what does that part do and why did you remove it?

Reply:
This was just some code added for the earlier tolerances implemented by @and-rewsmith . Since we don't need it anymore on direct method tests, I removed it. Original PR is here: https://github.com/Azure/iotedge/pull/5512/files